### ISSUE TITLE
Update plugin to use GCM Cocoapods <framework> reference in plugin.xml

### ISSUE THREAD

**Issue by chaffeqa** [EddyVerbruggen/cordova-plugin-googleplus#272]
Similar to what shazron did for [phonegap-push-plugin](https://github.com/phonegap/phonegap-plugin-push/pull/1183)

I'm gonna see if i can get one on our devs to contribute a pull request with this, but we are a small company with little bandwidth so I can't make any promises, sorry :(

**Comment by EddyVerbruggen** [comment]
Just a heads-up: the "problem" with cocoapods is that it's only supported with new cordova-ios versions so ppl will start running into issues once this lands.

**Comment by chaffeqa** [comment]
yeah, i think thought it would be nice to be ready to transition that way (have a pending pull request for when cordova-ios version drops, as well as allow those who would like to use nightlies to have a version of this plugin that work with that)

honestly its more `the sooner plugin owners can stop getting issues opened due to embedded frameworks the better`... i feel terrible for what you guys have to put up with, and i know you personally maintain a TON of great cordova plugins

**Comment by macdonst** [comment]
@EddyVerbruggen I still think we should be doing this across the board on plugins. There is supposed to be a feature in cordova where it picks the right version of the plugin depending on what version of the CLI you have installed. But of course there is currently a bug with that 😄

**Comment by christophmegusta** [comment]
hi Guys, how to handle this problem? I just added this plugin and now got this error too.
